### PR TITLE
Create updated AWS install in private preview

### PR DESCRIPTION
**Pull request by jwitz**
Based on https://www.notion.so/Draft-Docs-for-Cross-Account-Role-Update-adb9e84394574d2b98d147d6d137723f, this is an alternative to our other AWS installation doc

### CODE REVIEW

File: astro/cross-account-role-setup.md [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+id: 'cross-account-role-setup'
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+:::caution
+
+This feature is in [Private Preview](feature-previews.md) and is an alternative to the standard AWS installation process. For the standard installation steps, see [Install Astro on AWS](install-aws.md)
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:12]:
```suggestion
This feature is in [Private Preview](feature-previews.md) and is an alternative to the standard AWS installation process. For the standard installation, see [Install Astro on AWS](install-aws.md)
```

File: astro/cross-account-role-setup.md [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+
+:::
+
+To install Astro in a dedicated AWS account owned by your organization, you'll complete the following tasks:
+
+- Create an account on Astro.
+- Share AWS account information with Astronomer support.
+- Create a cross-account IAM role that Astro can assume within your new AWS account.
+
+Astronomer support will create a cluster within your AWS account that hosts the resources and Apache Airflow components necessary to deploy DAGs and execute tasks. If you'd like to support more than 1 Astro cluster, contact [Astronomer support](https://cloud.astronomer.io/support).
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:22]:
```suggestion
Astronomer support will create a cluster within your AWS account to host the resources and Airflow components necessary to deploy DAGs and execute tasks. If you need more than one Astro cluster, contact [Astronomer support](https://cloud.astronomer.io/support).
```

File: astro/cross-account-role-setup.md [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+
+## Step 1: Create an IAM role for Astronomer
+
+Astro requires an IAM role that delegates specific permissions for your data plane account to the Astro control plane. These permissions are required to manage the data plane and the data plane account itself.
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:26]:
```suggestion
Astro requires an IAM role that delegates specific permissions for your data plane account to the Astro control plane. These permissions are required to manage the data plane and the data plane account.
```

File: astro/cross-account-role-setup.md [hunk 4]
```diff
@@ -0,0 +1,289 @@
...
+
+See the following tables to learn more about each required permission and its purpose. To preview this role in its entirety, see its [CloudFormation template](https://astro-cross-account-role-template.s3.us-east-2.amazonaws.com/customer-account-prerelease.yaml).
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:28]:
```suggestion

```

Comment (kwcanuck) [reply astro/cross-account-role-setup.md:28]: This is unnecessary. Your introductory statements that precede the tables should inform users of their purpose and contents.

File: astro/cross-account-role-setup.md [hunk 5]
```diff
@@ -0,0 +1,289 @@
...
+
+### Permissions used to manage data plane resources
+
+As the Astro service changes, Astronomer may change the role policy attached to this cross account role. Astronomer will only make such changes after pre-announcing them via email. Please make sure you are subscribed to the [Status Page](https://status.astronomer.io/) to receive these alerts.
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:32]:
```suggestion
As the Astro service changes, Astronomer support might change the role policy attached to the cross account role. Astronomer support informs all organizations by email before implementing any changes. To make sure you receive these notifications, subscribe to the [Status Page](https://status.astronomer.io/). 
```
Google developer documentation style guide:

image

image

File: astro/cross-account-role-setup.md [hunk 6]
```diff
@@ -0,0 +1,289 @@
...
+
+Astro requires an IAM role that delegates specific permissions for your data plane account to the Astro control plane. These permissions are required to manage the data plane and the data plane account itself.
+
+See the following tables to learn more about each required permission and its purpose. To preview this role in its entirety, see its [CloudFormation template](https://astro-cross-account-role-template.s3.us-east-2.amazonaws.com/customer-account-prerelease.yaml).
+
+### Permissions used to manage data plane resources
+
+As the Astro service changes, Astronomer may change the role policy attached to this cross account role. Astronomer will only make such changes after pre-announcing them via email. Please make sure you are subscribed to the [Status Page](https://status.astronomer.io/) to receive these alerts. 
+
+To perform this function, Astro requires the following permissions to manage the cross account role.
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:34]:
```suggestion
The following table lists the permissions required by Astro to manage the cross account role.
```

File: astro/cross-account-role-setup.md [hunk 7]
```diff
@@ -0,0 +1,289 @@
...
+
+| Permissions                                            | Reason                                                                                                                        |
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:36]:
```suggestion
| Permission                                            | Reason                                                                                                                        |
```

Comment (kwcanuck) [reply astro/cross-account-role-setup.md:36]: If you don't like _Purpose_, maybe _Use_ would be more suitable?

File: astro/cross-account-role-setup.md [hunk 8]
```diff
@@ -0,0 +1,289 @@
...
+| ------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------- |
+| `cloudformation:*`                                     | Setup, teardown, and update the data plane.                                                                                   |
+| `eks:*`                                                | Manage EKS clusters.                                                                                                          |
+| `ec2:*`                                                | Manage compute and network infrastructure.                                                                                    |
+| `rds:*`                                                | Manage the backing RDS database.                                                                                              |
+| `s3:*`                                                 | Manage S3 buckets used by Astro clusters (for example, the bucket used for logging).                                          |
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:42]:
```suggestion
| `s3:*`                                                 | Manage S3 buckets used by Astro clusters. For example, the bucket used for logging.                                          |
```

See [Parentheses](https://developers.google.com/style/parentheses).

image

File: astro/cross-account-role-setup.md [hunk 9]
```diff
@@ -0,0 +1,289 @@
...
+| `sts:* `                                               | Manage tokens required for accessing AWS services.                                                                            |
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:43]:
```suggestion
| `sts:* `                                               | Manage the tokens required for accessing AWS services.                                                                            |
```

File: astro/cross-account-role-setup.md [hunk 10]
```diff
@@ -0,0 +1,289 @@
...
+
+To perform this function, Astro requires the following permissions to manage the cross account role.
+
+| Permissions                                            | Reason                                                                                                                        |
+| ------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------- |
+| `cloudformation:*`                                     | Setup, teardown, and update the data plane.                                                                                   |
+| `eks:*`                                                | Manage EKS clusters.                                                                                                          |
+| `ec2:*`                                                | Manage compute and network infrastructure.                                                                                    |
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:40]:
```suggestion
| `ec2:*`                                                | Manage the compute and network infrastructure.                                                                                    |
```

File: astro/cross-account-role-setup.md [hunk 11]
```diff
@@ -0,0 +1,289 @@
...
+| `rds:*`                                                | Manage the backing RDS database.                                                                                              |
+| `s3:*`                                                 | Manage S3 buckets used by Astro clusters (for example, the bucket used for logging).                                          |
+| `sts:* `                                               | Manage tokens required for accessing AWS services.                                                                            |
+| `autoscaling:*`                                        | Manage autoscaling policies for Astro infrastructure.                                                                         |
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:44]:
```suggestion
| `autoscaling:*`                                        | Manage the autoscaling policies for Astro infrastructure.                                                                         |
```

File: astro/cross-account-role-setup.md [hunk 12]
```diff
@@ -0,0 +1,289 @@
...
+| `cloudwatch:* , logs:* `                               | Monitor health of AWS resources.                                                                                              |
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:45]:
```suggestion
| `cloudwatch:* , logs:* `                               | Monitor the health of AWS resources.                                                                                              |
```

File: astro/cross-account-role-setup.md [hunk 13]
```diff
@@ -0,0 +1,289 @@
...
+| `kms:*`                                                | Manage keys used by Astro.                                                                                                    |
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:46]:
```suggestion
| `kms:*`                                                | Manage the keys used by Astro.                                                                                                    |
```

File: astro/cross-account-role-setup.md [hunk 14]
```diff
@@ -0,0 +1,289 @@
...
+| `secretsmanager:*`                                     | Store secrets used internally by Astro cluster.                                                                               |
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:47]:
```suggestion
| `secretsmanager:*`                                     | Store the secrets used internally by the Astro cluster.                                                                               |
```

File: astro/cross-account-role-setup.md [hunk 15]
```diff
@@ -0,0 +1,289 @@
...
+| `lambda:*`                                             | Scripts used to automate internal cluster operations (for example, to issue OIDC tokens to EKS clusters).                     |
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:48]:
```suggestion
| `lambda:*`                                             | Scripts used to automate internal cluster operations. For example, to issue OIDC tokens to EKS clusters.                     |
```

File: astro/cross-account-role-setup.md [hunk 16]
```diff
@@ -0,0 +1,289 @@
...
+| `route53:*`                                            | Resolve DNS names of services used by your data pipelines but running in other AWS accounts.                                  |
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:49]:
```suggestion
| `route53:*`                                            | Resolve the DNS names of the services used by your data pipelines but running in other AWS accounts.                                  |
```

File: astro/cross-account-role-setup.md [hunk 17]
```diff
@@ -0,0 +1,289 @@
...
+| `servicequotas:*`                                      | Monitor Service Quotas to ensure enough resource availability.                                                                |
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:50]:
```suggestion
| `servicequotas:*`                                      | Monitor Service Quotas to ensure resource availability.                                                                |
```

File: astro/cross-account-role-setup.md [hunk 18]
```diff
@@ -0,0 +1,289 @@
...
+| `ce:*`                                                 | Monitor costs.                                                                                                                |
+| `iam:Get* , iam:List* , iam:Tag*  , iam:Untag*`        | Enumerate and tag IAM objects, such as roles and policies.                                                                    |
+| `iam:OpenIDConnectProvider* `                          | Enable IAM Roles for Service Access.                                                                                          |
+| `iam:CreateRole iam:DeleteRole`                        | Create Operational Roles for Astro cluster. Note that this policy denies deletion of roles tagged with `customeraudit=TRUE` . |
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:54]:
```suggestion
| `iam:CreateRole iam:DeleteRole`                        | Create Operational Roles for the Astro cluster. This policy denies deletion of roles tagged with `customeraudit=TRUE` . |
```

File: astro/cross-account-role-setup.md [hunk 19]
```diff
@@ -0,0 +1,289 @@
...
+| iam:AttachRolePolicy , iam:PutRolePolicy , iam:Detach* | Create Operational Boundary and Permissions Policy for operational roles used by Astro clusters.                              |
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:55]:
```suggestion
| iam:AttachRolePolicy , iam:PutRolePolicy , iam:Detach* | Create the Operational Boundary and Permissions Policy for operational roles used by Astro clusters.                              |
```

File: astro/cross-account-role-setup.md [hunk 20]
```diff
@@ -0,0 +1,289 @@
...
+| iam:*InstanceProfile                                   | Manage instance profiles for cluster nodes.                                                                                   |
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:56]:
```suggestion
| iam:*InstanceProfile                                   | Manage the instance profiles for cluster nodes.                                                                                   |
```

File: astro/cross-account-role-setup.md [hunk 21]
```diff
@@ -0,0 +1,289 @@
...
+| iam:CreateServiceLinkedRole  iam:PassRole              | Manage internal roles used by AWS services.                                                                                   |
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:57]:
```suggestion
| iam:CreateServiceLinkedRole  iam:PassRole              | Manage the internal roles used by AWS services.                                                                                   |
```

File: astro/cross-account-role-setup.md [hunk 22]
```diff
@@ -0,0 +1,289 @@
...
+
+### Permissions used to manage the cross-account role
+
+As Astro changes, Astronomer may change the role policy attached to this cross account role. Astronomer will make changes to the role policy only after pre-announcing them through email. Subscribe to the [Status Page](https://status.astronomer.io/) to ensure that you receive these alerts.
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:61]:
```suggestion
As the Astro service changes, Astronomer support might change the role policy attached to the cross-account role. Astronomer support informs all organizations by email before implementing any changes. To make sure you receive these notifications, subscribe to the [Status Page](https://status.astronomer.io/). 
```

Comment (kwcanuck) [reply astro/cross-account-role-setup.md:61]: Make sure you're using _cross-account_ (hyphenated) consistently.

File: astro/cross-account-role-setup.md [hunk 23]
```diff
@@ -0,0 +1,289 @@
...
+| iam:AttachRolePolicy , iam:PutRolePolicy , iam:Detach* | Create Operational Boundary and Permissions Policy for operational roles used by Astro clusters.                              |
+| iam:*InstanceProfile                                   | Manage instance profiles for cluster nodes.                                                                                   |
+| iam:CreateServiceLinkedRole  iam:PassRole              | Manage internal roles used by AWS services.                                                                                   |
+
+### Permissions used to manage the cross-account role
+
+As Astro changes, Astronomer may change the role policy attached to this cross account role. Astronomer will make changes to the role policy only after pre-announcing them through email. Subscribe to the [Status Page](https://status.astronomer.io/) to ensure that you receive these alerts. 
+
+To perform this function, Astro requires the following permissions to manage the cross account role.
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:63]:
```suggestion
The following table lists the permissions required by Astro to manage the cross-account role.
```

File: astro/cross-account-role-setup.md [hunk 24]
```diff
@@ -0,0 +1,289 @@
...
+
+| Permissions                                                                               | Reason                                                |
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:65]:
```suggestion
| Permissions                                                                               | Purpose                                                |
```

File: astro/cross-account-role-setup.md [hunk 25]
```diff
@@ -0,0 +1,289 @@
...
+#### Monitor the cross-account role for changes (optional)
+
+To monitor changes to the cross-account role policy, Astronomer recommends setting up a CloudTrail in the data plane account and monitoring it for the following events: 
+
+| Event Names                              | Resource                                                         |
+| ---------------------------------------- | ---------------------------------------------------------------- |
+| `AttachRolePolicy , DetachRolePolicy`    | `roleName = astronomer-remote-management`                        |
+| `SetPolicyVersion , CreatePolicyVersion` | `policyArn = "arn:aws:iam::*:policy/AstronomerCrossAccountRole"` |
+
+To use CloudWatch to monitor this trail, follow the instructions in [Creating CloudWatch alarms for CloudTrail events](https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudwatch-alarms-for-cloudtrail.html) to apply the following filter to your CloudWatch metric:
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:78]:
```suggestion
To use a CloudWatch CloudTrail to changes to the cross-account role policy, see [Creating CloudWatch alarms for CloudTrail events](https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudwatch-alarms-for-cloudtrail.html).

Run the following command to apply a filter to your CloudWatch CloudTrail:
```

File: astro/cross-account-role-setup.md [hunk 26]
```diff
@@ -0,0 +1,289 @@
...
+
+```text
+{ ($.eventName = AttachRolePolicy || $.eventName = DetachRolePolicy || $.eventName = SetPolicyVersion || $.eventName = CreatePolicyVersion) && ($.requestParameters.policyArn = "*AstronomerCrossAccountRole"  || $.requestParameters.roleName = astronomer-remote-management) }
+```
+
+## Step 2: Access Astro
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:84]:
```suggestion
## Step 2: Access Astro
```

Where is Step 1?

File: astro/cross-account-role-setup.md [hunk 27]
```diff
@@ -0,0 +1,289 @@
...
+    - Enter your password and click **Continue**.
+    - To authenticate with an identity provider (IdP), click **Continue with SSO**, enter your username and password, and then click **Sign In**.
+    - To authenticate with your GitHub account, click **Continue with GitHub**, enter your username or email address, enter your password, and then click **Sign in**.
+    - To authenticate with your Google account, click **Continue with Google**, choose an account, enter your username and password, and then click **Sign In**.
+
+    If you're the first person in your Organization to authenticate, you'll be granted Organization owner permissions. You can create a Workspace and add other team members to the Workspace without the assistance of Astronomer support. See [Create a Workspace](manage-workspaces.md#create-a-workspace) and [Add a user](add-user.md). To integrate an identity provider (IdP) with Astro, see [Set up an identity provider](configure-idp.md).
+
+## Step 3:Retrieve an external ID from the Cloud UI
+
+You must be an Organization Owner to view the external ID. If you are not an Organization Owner, the field will not appear in the Cloud UI.
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:99]:
```suggestion
You must be an Organization Owner to view the external ID. If you are not an Organization Owner, the **AWS External ID** field will not appear in the Cloud UI.
```

File: astro/cross-account-role-setup.md [hunk 28]
```diff
@@ -0,0 +1,289 @@
...
+    aws iam attach-role-policy --policy-arn "$POLICY_ARN" --role-name "$ROLE_NAME"
+    
+    echo 'Setup complete'
+    ```
+
+</TabItem>
+</Tabs>
+
+
+## Step 4: Provide setup information to Astronomer
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:225]:
```suggestion
## Step 4: Provide setup information to Astronomer
```

Is Step 4 correct? Verify numbering.

File: astro/cross-account-role-setup.md [hunk 29]
```diff
@@ -0,0 +1,289 @@
...
+
+### VPC peering prerequisites (optional)
+
+If you need to VPC peer with Astronomer, provide the following information to your Astronomer representative:
+
+- Subnet CIDRs (RFC 1918 IP Space).
+- VPC Name/ID and region for peering with Astronomer. This is accessible through the [AWS VPC console](https://console.aws.amazon.com/vpc/).
+- The IPs of your DNS servers.
+
+## Step 4: Astronomer support creates the cluster
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:259]:
```suggestion
## Step 4: Astronomer support creates the cluster
```
Verify numbering.

File: astro/cross-account-role-setup.md [hunk 30]
```diff
@@ -0,0 +1,289 @@
...
+
+After you've created the cross-account IAM role for Astro, contact [Astronomer support](https://cloud.astronomer.io/support). Astronomer support will finish creating the cluster in your AWS account.
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:261]:
```suggestion
After you've created the cross-account IAM role for Astro, contact [Astronomer support](https://cloud.astronomer.io/support). Astronomer support creates the cluster in your AWS account.
```

In some instances, you're using _Cluster_ and in others, _cluster_. Use one style consistently.

File: astro/cross-account-role-setup.md [hunk 31]
```diff
@@ -0,0 +1,289 @@
...
+This process can take some time. Wait for confirmation from Astronomer support that the cluster has been created before creating a Deployment.
+
+If you submitted a VPC peering request, you'll need to accept the request from Astronomer after Astro is installed. To accept the request, see [Create a VPC peering connection](https://docs.aws.amazon.com/vpc/latest/peering/create-vpc-peering-connection.html).
+
+When VPC peering with Astronomer is complete, configure and validate the following items to ensure successful network communications between Astro and your resources:
+
+- Egress Routes on Astronomer Route Table
+- [Network ACLs](https://docs.aws.amazon.com/vpc/latest/userguide/vpc-network-acls.html#nacl-tasks) and/or [Security Group](https://docs.aws.amazon.com/vpc/latest/userguide/VPC_SecurityGroups.html#working-with-security-groups) rules of your resources
+
+## Next steps
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:272]:
```suggestion

```

File: astro/cross-account-role-setup.md [hunk 32]
```diff
@@ -0,0 +1,289 @@
...
+
+Congratulations on installing Astro on AWS! Astronomer recommends completing some of the following actions to make the most of your new installation.
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:274]:
```suggestion

```

File: astro/cross-account-role-setup.md [hunk 33]
```diff
@@ -0,0 +1,289 @@
...
+
+### Create a Deployment
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:276]:
```suggestion
### Create a Deployment
```
Should this be numbered? Should it be a H2?

File: astro/cross-account-role-setup.md [hunk 34]
```diff
@@ -0,0 +1,289 @@
...
+
+When Astronomer support confirms that your Astro cluster has been created, you can create a Deployment and start deploying DAGs. See [Create a Deployment](create-deployment.md). When you create your Deployment, the Astro cluster created by Astronomer support appears as an option in the **Cluster** list as shown in the following image.
+
+![Cloud UI New Deployment screen](/img/docs/create-new-deployment-select-cluster.png)
+
+
+### Additional documentation
+
+The following documents include setup steps for key Astro features and tools:
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:285]:
```suggestion

```

File: astro/cross-account-role-setup.md [hunk 35]
```diff
@@ -0,0 +1,289 @@
...
+Congratulations on installing Astro on AWS! Astronomer recommends completing some of the following actions to make the most of your new installation.
+
+### Create a Deployment
+
+When Astronomer support confirms that your Astro cluster has been created, you can create a Deployment and start deploying DAGs. See [Create a Deployment](create-deployment.md). When you create your Deployment, the Astro cluster created by Astronomer support appears as an option in the **Cluster** list as shown in the following image.
+
+![Cloud UI New Deployment screen](/img/docs/create-new-deployment-select-cluster.png)
+
+
+### Additional documentation
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:283]:
```suggestion
## Related documentation
```

**Review by kwcanuck — APPROVED** [review]
No related issue to resolve?

File: astro/cross-account-role-setup.md [hunk 36]
```diff
@@ -0,0 +1,336 @@
...
+
+- Create an account on Astro.
+- Share AWS account information with Astronomer support.
+- Create a cross-account IAM role that Astro can assume within your new AWS account.
+
+Astronomer support will create a cluster within your AWS account to host the resources and Airflow components necessary to deploy DAGs and execute tasks. If you need more than one Astro cluster, contact [Astronomer support](https://cloud.astronomer.io/support).
+
+## Prerequisites
+
+- A dedicated AWS account with minimum EC2 service quotas.
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:26]:
```suggestion
- A dedicated AWS account with the minimum EC2 service quotas.
```

File: astro/cross-account-role-setup.md [hunk 37]
```diff
@@ -0,0 +1,336 @@
...
+- A subscription to the [Astro Status Page](https://status.astronomer.io/). This will ensure that you're alerted in the case of an incident or scheduled maintenance.
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:27]:
```suggestion
- A subscription to the [Astro Status Page](https://status.astronomer.io/). This ensures that you'll be notified when there are incidents or when maintenance is scheduled.
```

File: astro/cross-account-role-setup.md [hunk 38]
```diff
@@ -0,0 +1,336 @@
...
+    - `https://cloud.astronomer.io/`
+    - `https://astro-<your-org>.datakin.com/`
+    - `https://<your-org>.astronomer.run/`
+    - `https://api.astronomer.io/`
+    - `https://images.astronomer.cloud/`
+    - `https://auth.astronomer.io/`
+    - `https://updates.astronomer.io/`
+    - `https://install.astronomer.io/`
+
+Astro requires a clean AWS account with a minimum set of EC2 service quotas. For security reasons, the install process is not currently supported on an AWS account that has other tooling running in it. For instructions on creating a new AWS account, follow [AWS documentation](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/).
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:39]:
```suggestion
Astro requires a clean AWS account with a minimum EC2 service quotas. For security reasons, the install process is not currently supported on an AWS account that has other tooling running in it. For instructions on creating a new AWS account, see [How do I create and activate a new AWS account?](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/).
```

File: astro/cross-account-role-setup.md [hunk 39]
```diff
@@ -0,0 +1,336 @@
...
+
+The required [EC2 service quotas](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-resource-limits.html) are:
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:41]:
```suggestion
The following table lists the required [EC2 service quotas](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-resource-limits.html).
```

File: astro/cross-account-role-setup.md [hunk 40]
```diff
@@ -0,0 +1,336 @@
...
+
+| QuotaCode  | QuotaName                                                        | Minimum Value  |
+| -----------| ---------------------------------------------------------------- | ---------------|
+| L-1216C47A | Running On-Demand Standard (A, C, D, H, I, M, R, T, Z) instances | 40             |
+| L-34B43A08 | All Standard (A, C, D, H, I, M, R, T, Z) Spot Instance Requests  | 40             |
+
+These are required to mitigate near term capacity risks and ensure a smooth onboarding experience on Astro. If you need to modify or increase a specific quota, see [Request a quota increase](https://docs.aws.amazon.com/servicequotas/latest/userguide/request-quota-increase.html).
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:48]:
```suggestion
These quotas are required to mitigate near term capacity risks and simplify the Astro onboarding experience. If you need to modify or increase a specific quota, see [Request a quota increase](https://docs.aws.amazon.com/servicequotas/latest/userguide/request-quota-increase.html).
```

File: astro/cross-account-role-setup.md [hunk 41]
```diff
@@ -0,0 +1,336 @@
...
+:::
+
+### VPC peering prerequisites (Optional)
+
+If any AWS resources are on a private network, you can choose between two options:
+
+- Allow traffic through the public internet and use allow-lists for communication.
+- Create a VPC Peering connection between the Astronomer VPC and the VPCs for your broader network.
+
+If you want to continue with the second option, you'll additionally need:
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:63]:
```suggestion
To implement the second option, you'll additionally need:
```

**Review by juliana-astro — APPROVED** [review]

File: astro/cross-account-role-setup.md [hunk 42]
```diff
@@ -0,0 +1,336 @@
...
+    aws iam attach-role-policy --policy-arn "$POLICY_ARN" --role-name "$ROLE_NAME"
+    
+    echo 'Setup complete'
+    ```
+
+</TabItem>
+</Tabs>
+
+
+## Step 4: Provide setup information to Astronomer
```
Comment (jwitz) [inline astro/cross-account-role-setup.md:272]:
```suggestion
## Step 5: Provide setup information to Astronomer
```

File: astro/cross-account-role-setup.md [hunk 43]
```diff
@@ -0,0 +1,336 @@
...
+Congratulations on installing Astro on AWS! Astronomer recommends completing some of the following actions to make the most of your new installation.
+
+### Create a Deployment
+
+When Astronomer support confirms that your Astro cluster has been created, you can create a Deployment and start deploying DAGs. See [Create a Deployment](create-deployment.md). When you create your Deployment, the Astro cluster created by Astronomer support appears as an option in the **Cluster** list as shown in the following image.
+
+![Cloud UI New Deployment screen](/img/docs/create-new-deployment-select-cluster.png)
+
+
+### Additional documentation
```
Comment (jwitz) [inline astro/cross-account-role-setup.md:328]:
```suggestion
### Read the documentation
```

File: astro/cross-account-role-setup.md [hunk 44]
```diff
@@ -0,0 +1,336 @@
...
+
+- Your AWS Account ID.
+- Your preferred Astro cluster name.
+- The AWS region that you want to host your cluster in.
+- Your preferred node instance type.
+- Your preferred maximum node count.
+
+If you do not specify configuration preferences, Astronomer creates a cluster with `m5.xlarge` nodes and a maximum node count of 20 in `us-east-1`. For information on all supported regions, configurations, and defaults, see [AWS cluster configurations](resource-reference-aws.md).
+
+To provision additional Clusters after completing your initial installation, see [Create a cluster](create-cluster.md).
```
Comment (jwitz) [inline astro/cross-account-role-setup.md:284]:
```suggestion
To provision additional clusters after completing your initial installation, see [Create a cluster](create-cluster.md).
```

**Pull request merged**